docs/DataDictionary.md

```diff
@@ -105,19 +105,19 @@ These are used for configuring a testing scenario.  `LoadTestConfig` will contai
 | EntityType      |   String   | Entity type used for filtering  | Yes | [`ClientStatus`, `LoadTestConfig`, `TestRun`] |
 | Id              |   String   | GUID used to retrieve the object directly. | Yes | |
 | Name            |   String   | Friendly name so that users may more easily identify configs | No | |
-| Files           |  String[]  | List of files to test   |   Yes     | match `--files` CLI flag |
+| Files           |  String[]  | List of files to test (default: `Null`)  |   Yes     | match `--files` CLI flag |
```

Comment:
I think this is for LodeRunner in client mode, and the default shouldn't be null because a value is required

Reply:
I removed (default: `Null`) for Files description.